### DISCUSSION TITLE
Cannot Get Over This Web3Auth Login Issue.

### DISCUSSION THREAD

**Discussion by gshdu** [Web3Auth/Web3Auth#791] [Help]
I have already overcome many issues with the web3auth login since I developed my Twitter Dapp. Now I'm facing a very strange message in the error which is the following:
![Screenshot 2022-10-28 234432](https://user-images.githubusercontent.com/98682949/198701557-a845408d-0314-4ca3-93a2-ac101f178df0.png)
I've contacted the live chat, but they couldn't really identify the problem. So, here I go, I'm expecting experts in this great discussion forum as it has a huge audience gathered.

**Comment by shahbaz17** [comment]
Hey @gshdu 

Can you share some of the information mentioned below?
* Which Web3Auth SDK & its version are you using?
* Please share the error in the console, along with the screenshot.
* Are you using any custom authentication? If so, please share your verifier details here.
